Makefile

```diff
@@ -167,7 +167,7 @@ bin/e2e-local.test: FORCE test/e2e/assets/chart/zz_chart.go
 
 # set go env and other vars, ensure that the dockerfile exists, and then build wait, cpb, and other command binaries and finally the kind image archive
 test/e2e-local.image.tar: export GOOS=linux
-test/e2e-local.image.tar: export GOARCH=386
+test/e2e-local.image.tar: export GOARCH=amd64
```

Comment:
Hi @perdasilva any way to pass the attributes from the Makefile or set them as env vars so the e2e test doesn't need to hard-code their values?

Reply:
Hey, I'm going to refactor the Makefile soon. I'm just making small changes now to make sure CI goes through. It's very finicky sometimes...